components/Collections/examples/CollectionsEditingExample.m

```diff
@@ -87,6 +92,32 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
   return cell;
 }
 
+- (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
+           viewForSupplementaryElementOfKind:(NSString *)kind
+                                 atIndexPath:(NSIndexPath *)indexPath
+{
+
+  // Must include below code snippet
+  id supplementaryView = [super collectionView:collectionView
```

Comment:
Can we s/id/UICollectionReusableView */ ?
I have a (semi-)rational distrust of id.